packages/typespec-test/test/eventgrid_modular/generated/typespec-ts/src/rest/isUnexpected.ts

```diff
@@ -28,11 +29,11 @@ const responseMap: Record<string, string[]> = {
 };
 
 export function isUnexpected(
-  response:
-    | PublishCloudEvent200Response
-    | PublishCloudEvents200Response
-    | PublishCloudEventDefaultResponse,
+  response: PublishCloudEvent200Response | PublishCloudEventDefaultResponse,
```

Comment:
what is causing the changes here ?

Reply:
bug here, see comments https://github.com/Azure/autorest.typescript/pull/2484#discussion_r1582501005